### DISCUSSION TITLE
Why am i not seeing all Labels on the Review Page

### DISCUSSION THREAD

**Discussion by cloudbr34k84** [blakeblackshear/frigate#13528] [General]
On the review page, im only seeing Cat Label, 
![image](https://github.com/user-attachments/assets/0d470131-c4e3-4048-ab6a-6be04fc457c1)

**Comment by NickM-27** [comment]
Because there are no review items for the other labels, so the filter would do nothing.

**Reply by cloudbr34k84** [reply]
i did think that but why show cat as the only label?
